docs/certificates.md

```diff
@@ -97,6 +97,24 @@ atmosphere_issuer_config:
    they `certificate` key includes the full chain in the correct order of issuer,
    intermediate(s) then root.
 
+## Using wildcard certificate
+
+If you have an existing wildcard certificate for all endpoints with Atmosphere,
+you can simply configure it as follows;
+
+- First, create a Kubernetes TLS secret using your wildcard certificate.
+  Details on how to create a TLS secret can be found[here](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets)
+
+  ```shell
+  kubectl -n openstack create secret tls wildcard-cert --key=./tls.key --cert=./tls.crt
+  ```
+
+- Second, set the above secret name as `ingress_wildcard_secret_name` ansible variable.
+
+  ```yaml
+  ingress_wildcard_secret_name: wildcard-cert
```

Comment:
Wouldn't it make sense to just use the openstack_helm_ingress_secret_name directly..?

Reply:
But it's defined in defaults so if the user overrides it, it will use their override. 

Perhaps the docs should be instead that you should disable cert manager annotation so that it doesn't try to overwrite it..?

Reply:
well, the annoying annotation is not coming from user's input. That is from this fixed var.

https://github.com/vexxhost/atmosphere/blob/5b9d0f1530389983ab03378e9d16143f380a8202/roles/openstack_helm_ingress/vars/main.yml#L15-L16

Reply:
What if the user sets openstack_helm_ingress_cluster_issuer: null ?

Reply:
I have set:
```
cluster_issuer_type: self-signed
openstack_helm_ingress_cluster_issuer: null
openstack_helm_ingress_secret_name: wildcard-cert

```
But the ingress gives me:

```
root@ctl1:~# kubectl -n openstack get ingress identity -o yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    cert-manager.io/cluster-issuer: ""
  creationTimestamp: "2023-02-01T11:48:20Z"
  generation: 2
  name: identity
  namespace: openstack
  resourceVersion: "161224"
  uid: 9fa5f73b-b222-48ea-a19e-2ce828a65981
spec:
  ingressClassName: atmosphere
  rules:
  - host: identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag
    http:
      paths:
      - backend:
          service:
            name: keystone-api
            port:
              number: 5000
        path: /
        pathType: Prefix
  tls:
  - hosts:
    - identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag
    secretName: wildcard-cert
status:
  loadBalancer:
    ingress:
    - ip: 10.101.240.200
```


And the ingress is not online:
```
root@ctl1:~# wget -O /dev/null https://identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag
--2023-02-01 16:38:06--  https://identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag/
Resolving identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag (identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag)... 10.9.14.81
Connecting to identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag (identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag)|10.9.14.81|:443... connected.
ERROR: cannot verify identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag's certificate, issued by ‘CN=Kubernetes Ingress Controller Fake Certificate,O=Acme Co’:
  Unable to locally verify the issuer's authority.
ERROR: no certificate subject alternative name matches
	requested host name ‘identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag’.
To connect to identity.10-9-14-81.ip.sslip.cloud.c3.cgm.ag insecurely, use `--no-check-certificate'.

```